css/style.css

```diff
@@ -24,6 +24,27 @@ a.btn:hover
     background-color: lightgray;
 }
 
+/* replaces <big> tag */
+em.big
+{
+    font-style: normal;
+    font-size: larger;
+}
+
+/* replaces <tt> tag */
+em.tt
+{
+    font-style: normal;
+    font-family: monospace;
+}
+
+/* replaces <u> tag */
+em.u
```

Comment:
I see. This macro is only used 6 times though, its usage should probably be replaced.